### PR TITLE
Remove email and address fields from `Person`

### LINKED ISSUE #28: Remove email and address fields

### PR DESCRIPTION
**Pull request by bnjmnt4n**
Closes #28.

### CODE REVIEW

File: src/main/java/socialite/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -38,15 +34,13 @@
...
-            + PREFIX_FACEBOOK + "john.doe"
-            + PREFIX_INSTAGRAM + "john.doe"
-            + PREFIX_TELEGRAM + "johndoe"
-            + PREFIX_TIKTOK + "johndoe"
-            + PREFIX_TWITTER + "johndoe";
+            + PREFIX_FACEBOOK + "john.doe "
+            + PREFIX_INSTAGRAM + "john.doe "
+            + PREFIX_TELEGRAM + "johndoe "
+            + PREFIX_TIKTOK + "johndoe "
+            + PREFIX_TWITTER + "johndoe ";
```
Comment (david-eom) [inline src/main/java/socialite/logic/commands/AddCommand.java:43]: Nice catch!

File: src/test/java/socialite/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -68,25 +66,11 @@ public void parse_invalidValue_failure() {
         CommandParserTestUtil.assertParseFailure(
                 parser, "1" + CommandTestUtil.INVALID_PHONE_DESC, Phone.MESSAGE_CONSTRAINTS
         );
-        // invalid email
```
Comment (david-eom) [inline src/test/java/socialite/logic/parser/EditCommandParserTest.java:71]: We should add some invalid insta/fb/tiktok handle test in the `EditCommandParserTest`, will set up issue for v1.3

File: src/test/java/socialite/logic/parser/EditCommandParserTest.java [hunk 3]
```diff
@@ -111,24 +95,21 @@ public void parse_invalidValue_failure() {
 
         // multiple invalid values, but only the first invalid value is captured
```
Comment (david-eom) [inline src/test/java/socialite/logic/parser/EditCommandParserTest.java:96]: I think the test here might not be indicative of "multiple invalid values".

**Review by david-eom — COMMENTED** [review]
Hi Ben, thanks for removing the two redundant fields. LGTM! I will do so later in the morning (preferably when one more team member skim through the changes). Realised that tests for `EditCommandParserTest` should be updated for different handles, we'll implement that in the upcoming iteration.

**Review by Zacchua — COMMENTED** [review]
Looks good thanks ben! And yup agree with xz that we should probably include more tests to increase coverage

**Pull request merged**